gpioioctl/example_test.go

```diff
@@ -0,0 +1,103 @@
+package gpioioctl_test
+
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"periph.io/x/conn/v3/driver/driverreg"
+	"periph.io/x/conn/v3/gpio"
+	"periph.io/x/conn/v3/gpio/gpioreg"
+	"periph.io/x/host/v3"
+	"periph.io/x/host/v3/gpioioctl"
+)
+
+func ExampleChips() {
+	_, _ = host.Init()
+	_, _ = driverreg.Init()
+
+	fmt.Println("GPIO Test Program")
+	chip := gpioioctl.Chips[0]
+	defer chip.Close()
+	fmt.Println(chip.String())
+	// Test by flashing an LED.
+	led := gpioreg.ByName("GPIO5")
+	fmt.Println("Flashing LED ", led.Name())
+	for i := range 20 {
+		_ = led.Out((i % 2) == 0)
+		time.Sleep(500 * time.Millisecond)
+	}
+	_ = led.Out(true)
+
+	testRotary(chip, "GPIO20", "GPIO21", "GPIO19")
```

Comment:
Caveat: Maybe, I'm not sure.
You may want to inline the function to make it look better. You can confirm that it looks good:
```
go install golang.org/x/pkgsite/cmd/pkgsite@latest
pkgsite
```
then navigate to http://localhost:8080/periph.io/x/host/v3

Reply:
I've fixed this and it shows in the format as expected.